docs/compile_readme_and_arrange_docs.py

```diff
@@ -93,6 +93,11 @@ def compile_readme():
         f"{abs_path}/sphinx/SphinxTutorial.ipynb",
     )
 
+    shutil.copy(
```

Comment:
What's the reason for the duplication?

Reply:
Unfortunately, sphinx cannot find files outside of it's source directory aside from autodocs.  Since this is not an installed module it needs to be copied.